### PR TITLE
Remove ReactAIContainer

### PR DESCRIPTION
**Pull request by pviotti**
ReactAIContainer is a singleton object that contains an instance of ReactAI and an instance of AppInsights to expose to the withAITracking function.  
I think we can avoid using it by exposing the AppInsights as a static variable of ReactAI.

This PR also 
- updates dependencies
- removes jest-environment-node-debug (because it's [not actively maintained](https://github.com/NikhilVerma/jest-environment-node-debug-fixed) - and anyway there are [better ways](https://github.com/Microsoft/vscode-recipes/tree/master/debugging-jest-tests) of debugging jest tests..)
- fixes a circular dependency warning upon build
- adds a VS Code launch file to debug jest tests
- simplifies tests

### CODE REVIEW

File: test/ReactAI.test.ts [hunk 1]
```diff
@@ -84,8 +84,8 @@ describe("ReactAI", () => {
 
     const pageViewTelemetry2 = { uri: "/home", properties: initialContext };
     const pageViewTelemetry3 = { uri: "/new-fancy-page", properties: initialContext };
-    expect(reactAI._aiInternal.trackPageView).toHaveBeenCalledTimes(2);
-    expect(reactAI._aiInternal.trackPageView).toHaveBeenNthCalledWith(1, pageViewTelemetry2);
-    expect(reactAI._aiInternal.trackPageView).toHaveBeenNthCalledWith(2, pageViewTelemetry3);
```
Comment (hiraldesai) [inline test/ReactAI.test.ts:89]: Can we call _aiInternal either “ai” or “appInsights”?

File: src/withAITracking.tsx [hunk 2]
```diff
@@ -54,7 +50,7 @@ export default function withAITracking<P>(Component: React.ComponentType<P>, com
         throw new Error("withAITracking:componentWillUnmount: mountTimestamp isn't initialized.");
       }
 
-      if (!container || !ai) {
```
Comment (hiraldesai) [inline src/withAITracking.tsx:57]: Is this cast necessary here?

**Review by hiraldesai — CHANGES_REQUESTED** [review]

File: src/withAITracking.tsx (hunk 2 above)
Comment (pviotti) [reply src/withAITracking.tsx:57]: no it's not: good catch, thanks

File: test/ReactAI.test.ts (hunk 1 above)
Comment (pviotti) [reply test/ReactAI.test.ts:89]: ok

**Pull request merged**